idmapping-rest/src/main/java/org/uniprot/api/idmapping/controller/request/uniprotkb/UniProtKBIdMappingBasicRequest.java

```diff
@@ -51,6 +49,20 @@ public class UniProtKBIdMappingBasicRequest extends IdMappingPageRequest {
             message = "{search.invalid.includeIsoform}")
     private String includeIsoform;
 
+    @Parameter(description = "Flag to write subsequences. Only accepted in fasta format")
+    @Pattern(
+            regexp = "true|false",
+            flags = {Pattern.Flag.CASE_INSENSITIVE},
+            message = "{search.invalid.subsequence}")
+    @ValidContentTypes(
+            contentTypes = {UniProtMediaType.FASTA_MEDIA_TYPE_VALUE},
+            message = "{search.invalid.contentType.subsequence}")
+    private String subSequence;
```

Comment:
I might miss it - but did you test
asking for sub sequence for any other format should not allowed